osCam/core/models.py

```diff
@@ -3,4 +3,4 @@
 from django.db import models
 from django.contrib.auth.models import User
 from django.utils import timezone
-from django.utils.translation import activate
+from django.utils.translation import activate
```

Comment:
is this a needed import? I don't see this page doing anything other than importing stuff.